src/components/markdown/ProblemsList/ProblemStatusCheckbox.tsx

```diff
@@ -9,21 +9,28 @@ import UserDataContext from '../../../context/UserDataContext/UserDataContext';
 import Tippy from '@tippyjs/react';
 import 'tippy.js/themes/light.css';
 import styled from 'styled-components';
-
+import feedbackModal from '../../feedbackModal';
 import MarkdownLayoutContext from '../../../context/MarkdownLayoutContext';
 import ConfettiContext from '../../../context/ConfettiContext';
+import Transition from '../../Transition';
+import { logError } from 'gatsby/dist/state-machines/develop/actions';
```

Comment:
what does this import do?

Reply:
Transition is for the modal, plus all I did was pull from the repo.

Reply:
I meant the logError thing. Transitoin import is fine

Reply:
I didnt write that

Reply:
So Idk...

Reply:
Should I remove?

Reply:
yes lol